Woof/Woof/View/SitterFlow/SitterProfileView/SitterProfileView.swift

```diff
@@ -0,0 +1,82 @@
+import SwiftUI
+
+struct SitterProfileView: View {
+    // MARK: - Internal interface
+
+    /// View model responsible to manage data from model layer.
+    @ObservedObject var viewModel = SitterProfileViewModel()
+
+    var body: some View {
+        VStack {
+            if isEditingMode {
+                VStack {
+                    EditSitterInformationView(
+                        name: $viewModel.name,
+                        surname: $viewModel.surname,
+                        phone: $viewModel.phone,
+                        bio: $viewModel.bio,
+                        pricePerHour: $viewModel.pricePerHour
+                    )
+
+                    HStack {
+                        Button("Save") {
+                            viewModel.save()
+                            isEditingMode.toggle()
+                        }
+
+                        Spacer()
+
+                        Button("Cancel") {
```

Comment:
When I click the "edit" button, change some data and click the "cancel" button, I expect that data will not be displayed in my profile. 
And this data is displayed. 
This confuses me.

Reply:
Yep, my fault, I fixed it. Check how it works now.